data/json/items/gun/shot.json

```diff
@@ -331,53 +391,183 @@
     "type": "GUN",
     "name": "single barrel shotgun",
     "description": "An old shotgun, possibly antique.  It is little more than a barrel, a wood stock, and a hammer to strike the cartridge.  Its simple design keeps it both light and accurate.",
-    "weight": "5216 g",
-    "volume": "2 L",
-    "price": 40000,
+    "weight": "2494 g",
+    "volume": "2044 ml",
+    "price": 10000,
     "to_hit": -1,
     "bashing": 12,
     "material": [ "steel", "wood" ],
-    "dispersion": 405,
+    "dispersion": 210,
     "durability": 9,
     "clip_size": 1,
-    "barrel_length": 4,
+    "barrel_length": 13,
     "valid_mod_locations": [
       [ "accessories", 4 ],
       [ "barrel", 1 ],
       [ "bore", 1 ],
       [ "muzzle", 1 ],
-      [ "sights", 1 ],
+      [ "mechanism", 1 ],
+      [ "sights mount", 1 ],
       [ "sling", 1 ],
-      [ "stock", 1 ],
+      [ "stock mount", 1 ],
       [ "grip mount", 1 ],
       [ "rail mount", 1 ],
       [ "underbarrel mount", 1 ]
     ],
     "flags": [ "NEVER_JAMS", "RELOAD_EJECT" ]
   },
   {
-    "id": "surv_levershotgun",
+    "id": "streetsweeper",
     "copy-from": "shotgun_base",
     "type": "GUN",
-    "name": "handmade lever shotgun",
-    "description": "A well designed homemade lever-action shotgun.  With a 8 round magazine, this is one of the better homemade weapons.",
-    "weight": "3934 g",
-    "volume": "2500 ml",
-    "price": 135000,
+    "name": "Cobray Streetsweeper",
+    "description": "Less shotgun and more comically oversized revolver, the Cobray Streetsweeper sold poorly before it was deemed a Destructive Device.  The cylinder is driven by a clockspring, cannot be indexed by hand, and must be ejected with an ejector rod.  Its unique design allows for all 12 shells to be fired in under 3 seconds, as demonstrated by the ATF technical branch.",
+    "weight": "419 g",
+    "volume": "3193 ml",
+    "looks_like": "revolver_shotgun",
+    "price": 120000,
+    "to_hit": -1,
+    "bashing": 8,
+    "material": [ "steel", "wood" ],
+    "dispersion": 855,
+    "durability": 6,
+    "clip_size": 12,
+    "reload": 220,
+    "barrel_length": 4,
+    "default_mods": [ "wire_stock", "grip" ],
+    "modes": [ [ "DEFAULT", "single", 1 ], [ "AUTO", "3-round", 3 ] ],
+    "valid_mod_locations": [
+      [ "accessories", 4 ],
+      [ "barrel", 1 ],
+      [ "bore", 1 ],
```

Comment:
Caliber conversion kits being un-obsoleted in the future confirmed? 🎉

Reply:
Mostly just an artifact of judicious copy and pasting.